security/pkg/pki/util/keycertbundle.go

```diff
@@ -332,6 +342,20 @@ func extractCertExpiryTimestamp(certType string, certPem []byte) (float64, error
 	return expiryTimestamp, nil
 }
 
+func extractCertExpirySeconds(certType string, certPem []byte) (float64, error) {
```

Comment:
One idea for testing this method could be to verify that the expirySeconds returned is greater than 0 and less than "test start time - cert.NotAfter".

Reply:
I will add a follow up PR adding tests.